src/js/error.rs

```diff
@@ -1 +1,103 @@
-pub use internal::js::error::{throw, JsError, Kind};
+// pub use internal::js::error::{throw, JsError, Kind};
```

Comment:
Are we going to leave these in here commented out?

Reply:
Maybe these caused git not to recognize the file name changes.

Reply:
D'oh, yeah, I forgot to delete those -- will do!